plugins/source/azure/client/client.go

```diff
@@ -154,6 +167,8 @@ func New(ctx context.Context, logger zerolog.Logger, s specs.Source, _ source.Op
 		return nil, err
 	}
 
+	log.SetListener(nil)
```

Comment:
Removing the listener would also ignore any EventAuthentication error occuring after the initial setup, like a fail to refresh token etc.

Reply:
Those are returned as errors from functions. We didn't set the listener before so we're not missing on anything here